api-gateway/src/main/resources/static/scripts/visits/visits.controller.js

```diff
@@ -682,7 +730,73 @@ angular.module('visits')
             }, function () {
                 console.log("Failed to create corresponding bill!");
             });
-        }
+        }*/
+
+        self.someFunction = function() {
+            console.log('Using ownerId in someFunction:', self.ownerId);
+            console.log('Using petId in someFunction:', self.petId);
+        };
+        $scope.goBack = function() {
```

Comment:
[nitpick] Same thing here, the name of the function could be a little more descriptive of it's function. Therefore : goBackToVisitsList